src/sagemaker_containers/__init__.py

```diff
@@ -12,6 +12,7 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
+from sagemaker_containers import modules
```

Comment:
Nitpick - does importing modules here change anything? A user would still refer to it as sagemaker_containers.modules, right?

And for adding it to __all__, my understanding is that this controls the "from sagemaker_containers import *" behavior. In this case, we don't really want "modules" to be imported into the current user's namespace as is, do we?

Reply:
It is actually not a nitpick @winstonaws. We need to keep modules so we can import like this:
```python 
import sagemaker-containers as smc

smc.modules.something
```

You are correct that ```__all__``` imports everything. I am removing it.